sunpy/map/header_helper.py

```diff
@@ -110,10 +117,10 @@ def make_fitswcs_header(data, coordinate, reference_pixel: u.pix = None,
     if isinstance(coordinate, frames.Heliocentric):
         raise ValueError("This function does not currently support heliocentric coordinates.")
 
-    meta_wcs = _get_wcs_meta(coordinate)
+    meta_wcs = _get_wcs_meta(coordinate, projection_code)
 
     if hasattr(coordinate, "observer") and isinstance(coordinate.observer, frames.BaseCoordinateFrame):
-        meta_observer = _get_observer_meta(coordinate)
+        meta_observer = get_observer_meta(coordinate.observer, coordinate.rsun)
```

Comment:
is there a reason that coordinate.rsun is pulled out here? just curious why the coordinate isn't just passed

Reply:
The main reason I wanted to do this was to make the `get_observer_meta` useful in isolation. It might be occasionally useful to convert a SkyCoord for observer to header keys, for instance in the differential rotation code etc.